doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -146,37 +189,77 @@ across multiple tenant’s environments.
 
 #### 5.3.9 3rd-party test tool integration
 
-Allows executing the test cases from multiple 3rd party test tools and producing
-the results
+In any telco environment, heterogeneous is inevitable and need for executing the
+test cases from different test tools is required. so system would support to
+on-board the 3rd party test tools and corresponding test cases from them and
+allows executing the test cases from those 3rd party test tools.
 
 #### 5.3.10 Certification management & integration
 
 Platform should have integration with OVP certification portal for submitting
-results with OVP defined format
-
-#### 5.3.11 OVP portal integration
+results with OVP defined format.
 
-#### 5.3.12 User & System interfaces
+It should enable repository of certified VNFs which can be used for testing
+validation and performance.
 
-User interface:
+#### 5.3.11 User & System interfaces
 
->   CLI
+**User interface**:
 
->   Web portal
+*   CLI
+*   Web portal
 
-Programming interface: REST API
+**Programming interface**:
 
-#### 5.3.13 Deliverables
+* REST API
+* gRPC
 
-Docker image based installation
+#### 5.3.12 Deliverables
+Platform should be able to get deployed in both container and cloud environments.
+so following model deliverables would enable it:
 
-Standalone installation scripts and zip artifact
+* Docker image based installation
+* Standalone installation scripts and zip artifact
 
 <a name="5.4"></a>
-### 5.4 Certification Process
+### 5.4 Certification Methodology
+Perform the required VNF certtification based on scope defined in this chapter and by
+leveraging the test cases and tools which provides the features defined under
+platform requirements as above.
 
-This process details the end-end steps required for certifying the given VNF by
-using ths platform (providing above requirements), producing the test results to
+It defines the end-end Certification Process required for certifying the given VNF by
+using this platform (providing above requirements), producing the test results to
 certification portal , review and granting the badge to this VNF.
 
+[More details](https://wiki.lfnetworking.org/display/LN/VNF+Certification+Lifecycle)
+
 #### 5.4.1 Entry & Exit criteria
+
+**Entry criteria**: Before entering into the VNF certification, VNF needs to
+satisfy the following requirements as entry pass:
+
+* *Environment Requirements* : Details on required dependent system which are
+required to deploy and test VNF.
+* *VNF artifact* : VNF cloud (native) image, VNF configurations and guidelines,

```

Comment:
Can we also add a requirement for VNF deployment script/automation here ?

Reply:
done

Reply:
@msidana - please review to see if we can close